source/guide_gitea.rst

```diff
@@ -96,93 +157,276 @@ and a new config file at ``~/gitea/custom/conf/app.ini`` to specify the port and
+  
+  [database]
+  DB_TYPE           = mysql
+  HOST              = 127.0.0.1:3306
+  NAME              = isabell_gitea
+  USER              = isabell
+  PASSWD            = <MySQL_PASSWORD>
+  SSL_MODE          = disable
+  
+  [security]
+  INSTALL_LOCK          = true
+  MIN_PASSWORD_LENGTH   = 8
+  PASSWORD_COMPLEXITY   = lower,digit
+  PASSWORD_HASH_ALGO    = pbkdf2 ; [pbkdf2, argon2, scrypt, bcrypt]
+  LOGIN_REMEMBER_DAYS   = 5
+  CSRF_COOKIE_HTTP_ONLY = true
+  INTERNAL_TOKEN        = 
+  SECRET_KEY            = <RANDOM_32_CHARS>
+  
+  [service]
+  DISABLE_REGISTRATION                   = true
+  SHOW_REGISTRATION_BUTTON               = false
+  REQUIRE_EXTERNAL_REGISTRATION_PASSWORD = false
+  ALLOW_ONLY_EXTERNAL_REGISTRATION       = false
+  REQUIRE_SIGNIN_VIEW                    = true
+  ENABLE_CAPTCHA                         = false
+  REGISTER_EMAIL_CONFIRM                 = true
+  ACTIVE_CODE_LIVE_MINUTES               = 60
+  RESET_PASSWD_CODE_LIVE_MINUTES         = 60
+  ENABLE_NOTIFY_MAIL                     = false
+  SHOW_MILESTONES_DASHBOARD_PAGE         = true
+  DEFAULT_ALLOW_CREATE_ORGANIZATION      = true
+  DEFAULT_ORG_VISIBILITY                 = private ; [public, limited, private]
+  DEFAULT_ORG_MEMBER_VISIBLE             = false
+  DEFAULT_ENABLE_TIMETRACKING            = true
+  DEFAULT_KEEP_EMAIL_PRIVATE             = true
+  NO_REPLY_ADDRESS                       = noreply.isabell.uber.space
+  
+  [repository]
+  ROOT = /home/isabell/gitea/repositories
+  DEFAULT_PRIVATE = private ; [last, private, public]
+
+  [ui]
+  THEMES        = gitea,arc-green
+  DEFAULT_THEME = arc-green
+
+  [mailer]
+  ENABLED     = true
+  MAILER_TYPE = sendmail
+  FROM        = isabell@uber.space
+  
+  [time]
+  FORMAT              = 2006-01-02 15:04:05
+  DEFAULT_UI_LOCATION = Europe/Amsterdam
+  
+  [log]
+  MODE      = file
+  LEVEL     = info ; [Trace, Debug, Info, Warn, Error, Critical, Fatal, None]
+  ROOT_PATH = /home/isabell/gitea/log
+  
+  [log.file]
+  LOG_ROTATE   = true
+  DAILY_ROTATE = true
+  MAX_DAYS     = 14
+  COMPRESS     = false
+
+Gitea using external renderer (optional)
+----------------------------------------
+
+We can install an extra `external rendering <https://docs.gitea.io/en-us/external-renderers/>`_ extension AsciiDoc.
 
-Setup daemon
-------------
+.. code-block:: console
+  
+  [isabell@stardust ~]$ gem install asciidoctor
+  Fetching asciidoctor-2.0.10.gem
+  WARNING:  You don't have /home/isabell/.gem/ruby/2.7.0/bin in your PATH,
+  	  gem executables will not run.
+  Successfully installed asciidoctor-2.0.10
+  1 gem installed
+  [isabell@stardust ~]$
 
-Create ``~/etc/services.d/gitea.ini`` with the following content:
+.. note:: Don't be irritated by the warning that the bin folder isn't in the path. Uberspace is taking care of it and provide it via ``/opt/uberspace/etc/isabell/binpaths/ruby/asciidoctor``. You can check with ``[isabell@stardust ~]$ which asciidoctor``.
```

Comment:
`/opt/uberspace/etc/isabell/binpaths` is not documented publicly. While we don't consider this a secret (it's right there in front of you ;)), we try to support details, that are documented publicly, as long as possible. If not necessary we try to not include too many implementation details outside of the "background" section in our manual.

Please shorten this to "Uberspace is taking care of it. You can check with (...)".

Reply:
Done.